docs/src/dev-guide/building-package.md

```diff
@@ -13,8 +13,7 @@ prebuilt version instead, check out the [releases](https://github.com/y-scope/cl
 * Python 3.9 or newer
 * python3-dev
 * python3-venv (for the version of Python installed)
-* [Task] >= 3.40.0 and < 3.43.0
-  * We constrain the version due to unresolved [issues][clp-issue-872].
+* [Task] >= 3.40.0
```

Comment:
```suggestion
* [Task] 3.40.0
```

Reply:
as per the above installation script, shall we write
```suggestion
* [Task] 3.44.0
```